website/content/en/preview/getting-started/getting-started-with-terraform/_index.md

```diff
@@ -108,42 +108,57 @@ module "vpc" {
   single_nat_gateway     = true
   one_nat_gateway_per_az = false
 
+  public_subnet_tags = {
+    "kubernetes.io/cluster/${local.cluster_name}" = "shared"
+    "kubernetes.io/role/elb"                      = 1
+  }
+
   private_subnet_tags = {
-    "kubernetes.io/cluster/${local.cluster_name}" = "owned"
-    # Tags subnets for Karpenter auto-discovery
-    "karpenter.sh/discovery" = local.cluster_name
+    "kubernetes.io/cluster/${local.cluster_name}" = "shared"
+    "kubernetes.io/role/internal-elb"             = 1
   }
 }
 
 module "eks" {
   # https://registry.terraform.io/modules/terraform-aws-modules/eks/aws/latest
   source  = "terraform-aws-modules/eks/aws"
-  version = "18.17.0"
+  version = "18.20.5"
 
   cluster_name    = local.cluster_name
-  cluster_version = "1.21"
+  cluster_version = "1.22"
 
   vpc_id     = module.vpc.vpc_id
   subnet_ids = module.vpc.private_subnets
 
   # Required for Karpenter role below
   enable_irsa = true
 
-  # We will rely only on the cluster security group created by the EKS service
-  # See note below for `tags`
-  create_cluster_security_group = false
-  create_node_security_group    = false
+  node_security_group_additional_rules = {
+    ingress_nodes_karpenter_port = {
+      description                   = "Cluster API to Node group for Karpenter webhook"
+      protocol                      = "tcp"
+      from_port                     = 8443
+      to_port                       = 8443
+      type                          = "ingress"
+      source_cluster_security_group = true
+    }
+  }
+
+  node_security_group_tags = {
+    # NOTE - if creating multiple security groups with this module, only tag the
+    # security group that Karpenter should utilize with the following tag
+    # (i.e. - at most, only one security group should have this tag in your account)
+    "karpenter.sh/discovery/${local.cluster_name}" = local.cluster_name
+  }
 
   # Only need one node to get Karpenter up and running.
   # This ensures core services such as VPC CNI, CoreDNS, etc. are up and running
   # so that Karpetner can be deployed and start managing compute capacity as required
   eks_managed_node_groups = {
     initial = {
-      instance_types = ["t3.medium"]
-      # We don't need the node security group since we are using the
-      # cluster-created security group, which Karpenter will also use
-      create_security_group                 = false
-      attach_cluster_primary_security_group = true
+      instance_types = ["t3.large"]
```

Comment:
In the other guide, we use m5.large: https://karpenter.sh/v0.10.1/getting-started/getting-started-with-eksctl/

Is there a reason we wanted to stick to t3 here? Pretty small detail, but I'd like to sync guides where easily possible.

Reply:
No reason in particular - I will update to align